code/example-project/lib/modules/foods/index.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    extend: 'apostrophe-pieces',
+    name: 'foods',
+    label: 'Food',
+    pluralLabel: 'Food',
+    addFields: [
+    {
+            name: 'title',
```

Comment:
Inconsistent formatting. This project should have an eslintrc file, taken from one of our client projects, with the appropriate dependencies in package.json, and then "npx eslint . --fix" will fix most of this for you.